index_v1.html

```diff
@@ -1,42 +1,101 @@
+<?php
+$currentPage = 'login_1';
+include 'nav-login.php';

```

Comment:
**Including Navigation:** The line include 'nav-login.php'; brings in the navigation elements for the login page. This allows for a consistent navigation experience across different parts of the site.